models/src/main/java/com/vimeo/networking2/Live.kt

```diff
@@ -69,9 +69,8 @@ data class Live(
 
     /**
      * If [status] is [LiveStatus.STREAMING_ERROR], this is the reason for that error.
-     *
-     * Requires [CapabilitiesType.CAPABILITY_LIVE_EVENT].
      */
+    @Internal
```

Comment:
if the class itself is annotated `@Internal` should we also be annotating the properties of the class?

Reply:
Yeah. I update the PR to mark the fields internal as well.